dashboard/app/models/user.rb

```diff
@@ -2660,6 +2667,12 @@ def code_review_groups
 
   # Values for the `child_account_compliance_state` attribute
   module ChildAccountCompliance
+    # The student's account does not have compliance.
+    ACCOUNT_CREATED = 'c'.freeze
```

Comment:
I don't use this... this would be in the new account flow. @daynew, is this what is expected? Is it duplicated work?

Reply:
I know it's in the tech design, but as we have implemented things, I have come to realize we can get away with not having that state. You can remove it from your PR. I will think about adding it back in a separate one.

Reply:
I will push a commit that removes it.